src/main/java/frc/robot/commands/LiftToHeight.java

```diff
@@ -25,6 +23,25 @@ public LiftToHeight(HangSubsystem hangSubsystem) {
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
-    hangSubsystem.liftToHeight();
+    hangSubsystem.resetEncoder();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    hangSubsystem.liftLift();
```

Comment:
Maybe we rename this?

```suggestion
    hangSubsystem.raiseLift();
```